### DISCUSSION TITLE
ı have a error

### DISCUSSION THREAD

**Discussion by 0ops5** [Kav-K/GPTDiscord#214] [Q&A] [answered]
Loading environment from .env
Loading environment from /root/GPT3Discord/etc/environment
Loading environment from None
Attempting to retrieve the settings DB
Retrieved the settings DB
Building language detector
Language detector built
CHAT_BYPASS_ROLES is not defined properly in the environment file!Please copy your server's role and put it into CHAT_BYPASS_ROLES in the .env file.For example a line should look like: `CHAT_BYPASS_ROLES="bypass"`
Defaulting to allowing NO ONE to bypass chat moderation
Attempting to retrieve the General and Moderations DB
Retrieved the General and Moderations DB
Attempting to retrieve the General and Moderations DB
Building language detector
Language detector built
Traceback (most recent call last):
  File "/root/GPT3Discord/gpt3discord.py", line 22, in <module>
    from cogs.index_service_cog import IndexService
  File "/root/GPT3Discord/cogs/index_service_cog.py", line 10, in <module>
    from models.index_model import Index_handler
  File "/root/GPT3Discord/models/index_model.py", line 59, in <module>
    RemoteDepthReader = download_loader("RemoteDepthReader")
  File "/usr/local/lib/python3.9/dist-packages/llama_index/readers/download.py", line 190, in download_loader
    pkg_resources.require([str(r) for r in requirements])
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 891, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 782, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (tqdm 4.65.0 (/usr/local/lib/python3.9/dist-packages), Requirement.parse('tqdm==4.64.1'))

**Comment by Kav-K** [comment]
Install tqdm==4.64.1 with `pip install tqdm==4.64.1` and it should be fixed

@Hikari-Haru Do you know why we're having this? Is it from some package overriding?

**Reply by 0ops5** [reply]
I don't know, I'm just so new at this stuff. I followed the installation instructions of the droplette that I created on digitalocean.
`pip install tqdm==4.64.1` this worked for me. thanks have a good day...

**Comment by cherryroots** [comment, marked as answer]
The llama index download loaders have extremely narrow package requirements, so some package we install installs tqdm, but it's 4.65, the loaded reader wants specifically 4.64.1 so it throws. We have thus version specified, but only in the full requirements. So move it from full to base, and add it to the normal requirements 

It's this requirements.txt that's throwing up trouble https://github.com/emptycrown/llama-hub/tree/main/loader_hub/remote_depth
